drivers/hmis_external_apis/app/models/hmis_external_apis/oauth_client_connection.rb

```diff
@@ -55,6 +76,14 @@ def request(verb, url, payload = nil)
           raise "invalid verb #{verb}"
         end
 
+      if result
+        request_log.update!(
+          content_type: result.content_type,
+          response: result.body,
+          http_status: result.status,
```

Comment:
I'm getting an error here, `http_status` is not a column on ExternalRequestLog. Fixing here https://github.com/greenriver/hmis-warehouse/pull/3150

Reply:
@gigxz that's odd, I see that column locally. It's also defined in `db/warehouse_structure.sql`
```ruby
HmisExternalApis::ExternalRequestLog.column_names.grep /http/
=> ["http_method", "http_status"]
```

Maybe there's some inconsistency in our migrations?

Reply:
Oh weird, I'll check it out. It wasn't in my structure file, but I had already run a migration, so maybe it got dropped.

Reply:
Thanks for catching that. I don't know why that happened, but I added it back locally for me. The column is present in the staging environment.